putsurface-wayland compile error

putsurface_wayland fails to compile on Fedora if libwayland-server-devel is not installed:

`putsurface_wayland.c:34:28: fatal error: wayland-server.h: No such file or directory
#include <wayland-server.h>`

On Fedora, the client and server development headers and libraries are provided by separate packages (Ubuntu provides both in one package).  On Fedora, wayland-client.h header is provided by libwayland-client-devel package.

putsurface_wayland is a wayland client application and should not be including wayland-server.h... instead, it should include wayland-client.h.  Also, configure.ac only checks for libwayland-client library, which is correct, thus we should not be using wayland-server files in source.

--- a/putsurface/putsurface_wayland.c
+++ b/putsurface/putsurface_wayland.c
@@ -31,7 +31,7 @@
 #else
 # include <va/va_wayland.h>
 #endif
-#include <wayland-server.h>
+#include <wayland-client.h>
 
 static void *open_display(void);
 static void close_display(void *win_display);

